.asf.yaml

```diff
@@ -0,0 +1,54 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# See: https://cwiki.apache.org/confluence/display/INFRA/git+-+.asf.yaml+features
+
+github:
+  description: "Apache Kyuubi is a distributed multi-tenant JDBC server for large-scale data processing and analytics, built on top of Apache Spark"
```

Comment:
I'm not sure if the **Apache Kyuubi** without **(Incubating)** meets the Apache trademark requirements.

Reply:
These rules are generalized to describe ASF software associated with the trademark "Apache ProjectName", or more generally "ProjectName" when it is understood to refer to our specific Apache ProjectName software product. Like most ASF software, this ProjectName software is maintained by the Apache ProjectName project, or by the ProjectName sub-project of another project, such as the "Apache Incubator" (itself an ASF trademark).

https://incubator.apache.org/projects/